### PR TITLE
Fix index pattern without time field tests

### PR DESCRIPTION
**Pull request by wanglam**
### Description

Fix index pattern without time field tests failed when security enabled, this test was failed in [main branch](https://github.com/opensearch-project/opensearch-dashboards-functional-test/actions/runs/7552595626/job/20765731545?pr=999), see screenshot:
image

These tests can be passed in the [workflow](https://github.com/opensearch-project/opensearch-dashboards-functional-test/actions/runs/7782381635/job/21218686024?pr=1054), here is the screenshot:
image

Also add `CYPRESS_NO_COMMAND_LOG` to the workflow file, unhang OSD Core workflows to bypass #1055 

### Issues Resolved

[List any issues this PR will resolve]

### Check List

- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/workflows/cypress-workflow-bundle-snapshot-based.yml [hunk 1]
```diff
@@ -33,6 +33,6 @@ jobs:
     uses: ./.github/workflows/release-e2e-workflow-template.yml
     with:
       test-name: Core Dashboards using Bundle Snapshot
-      test-command: env CYPRESS_ML_COMMONS_DASHBOARDS_ENABLED=true CYPRESS_VISBUILDER_ENABLED=true CYPRESS_DATASOURCE_MANAGEMENT_ENABLED=true yarn cypress:run-without-security --browser chromium --spec 'cypress/integration/core-opensearch-dashboards/opensearch-dashboards/**/*.js'
+      test-command: env CYPRESS_NO_COMMAND_LOG=1 CYPRESS_ML_COMMONS_DASHBOARDS_ENABLED=true CYPRESS_VISBUILDER_ENABLED=true CYPRESS_DATASOURCE_MANAGEMENT_ENABLED=true yarn cypress:run-without-security --browser chromium --spec 'cypress/integration/core-opensearch-dashboards/opensearch-dashboards/**/*.js'
```
Comment (ruanyl) [inline .github/workflows/cypress-workflow-bundle-snapshot-based.yml:36]: Did you include this change to this PR intentionally? Seems not related to this PR

Comment (wanglam) [reply .github/workflows/cypress-workflow-bundle-snapshot-based.yml:36]: Yeah, just bypass the [pending issue](https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/1055). The workflow will always pending without this env variable.

Comment (wanglam) [reply .github/workflows/cypress-workflow-bundle-snapshot-based.yml]: Removed.

**Review by raintygao — APPROVED** [review]

**Review by Hailong-am — APPROVED** [review]

**Comment by Hailong-am** [comment]
@wanglam could you help to resolve the conflicts?

**Comment by wanglam** [comment]
The functional tests were successful in current main branch. Here are the job details: https://github.com/opensearch-project/opensearch-dashboards-functional-test/actions/runs/8828305033?pr=999 . I think we don't need to change the test cases anymore. I will close this PR.
